pyserini/prebuilt_index_info.py

```diff
@@ -151,5 +151,65 @@
         "documents": 4170312,
         "unique_terms": -1,
         "downloaded": False
+    },
+    "trec-covid5": {
+        "description": "Trec Covid Round 5",
+        "urls": [
+            "https://www.dropbox.com/s/9hfowxi7zenuaay/lucene-index-cord19-abstract-2020-07-16.tar.gz?dl=1"
+        ],
+        "md5": "c883571ccc78b4c2ce05b41eb07f5405",
+        "size compressed (bytes)": 2796524,
+        "total_terms": 22100404,
+        "documents": 192459,
+        "unique_terms": 195875,
+        "downloaded": False
+    },
+    "trec-covid4": {
```

Comment:
For all rounds, we should have three separate indexes, like above.

Reply:
All full-text and paragraph pre-built index are added